source/_includes/puppet-collections/_puppet_collections_intro.md

```diff
@@ -2,15 +2,31 @@
 
-The collections are organized into two tiers that map to the Puppet Enterprise release trains which are "downstream" from the open-source components in the collection. Numbered collections (like PC1) are long-lived, stable repositories that map to Long Term Support (LTS) Puppet Enterprise releases; the standard support PE releases will track a collection named `latest` ([The Puppet Enterprise Lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle) page has more detail about the PE release trains). A collection repository will support operating system-specific packages for the OSes available when it's launched, until thirty days past the end of their vendor-determined lifespan.
+Collection repositories are organized into two tiers that correspond to Puppet Enterprise releases, which are downstream from the collection's open-source components:
 
-Each collection is opt-in, and if you use `ensure => latest` to manage your Puppet packages, you'll stay current with updated packages in the collection you're using. A numbered collection will keep the same major version of each component package for its lifetime; backwards-incompatibilities will be minimized but so will feature development. The `latest` collection will get every new version of the component packages as they're released, so using it will keep you current. Be aware, though, that it will receive major-version updates which can introduce backwards incompatibilities. We recommend tracking the `latest` collection but keeping your infrastructure "pinned" to known versions. For example, if you're using the [puppetlabs-puppet_agent](https://forge.puppet.com/puppetlabs/puppet_agent) module to manage the installed version of the all-in-one agent, you could use:
+-   **Numbered collections, such as Puppet Collection 1 (PC1),** are long-lived, stable repositories from which Long-Term Support (LTS) Puppet Enterprise releases are built. Numbered collections maintain the same major version of each component package during its lifetime, which delivers bug fixes and minimizes breaking changes, but also introduces fewer new features.
+-   **The "latest" collection** follows every release of Puppet Enterprise, including versions not considered LTS releases, and is updated with new major-version releases that might introduce breaking changes.
+
+Puppet publishes updates for operating systems starting from the time a package is first published for that operating system to a collection repository, and stops updating those packages 30 days after the end of the operating system's vendor-determined lifespan.
+
+See [The Puppet Enterprise Lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle) for information about PE release trains.
```

Comment:
I'm not sure about the use of "release trains" here, but I don't know what the new lifecycle document looks like.